tasks/Download BAM/Download Any BAM.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
<p>are we supposed to be able to see the output?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
It looks good, is this supposed to have output data in the journal? or is the user supposed to run it themselves and follow along? @mckinsel i know its been a while sense you have modified the file, any input?

If they are to be following along i think it makes sense to test if this file contains output; so the next time someone updates it they don't commit the history, if its to be omitted. @natanlao

Reply:
Okay, then I'll update this notebook to include output, and then file a separate PR to check for notebook output in CI. Does that sound ok?